.gitignore

```diff
@@ -63,3 +65,8 @@
 
 .idea/
 cmake-build-debug/
+
+# do you use ninja for any of the plugins?
```

Comment:
Is this a question for the review, or should it stay? Maybe rephrase to `ignore ninja build files` to make it clear?